app/assets/scripts/components/map/measurements-layer.js

```diff
@@ -102,43 +185,58 @@ export default function MeasurementsLayer({
         .setLngLat(coordinates)
         .setDOMContent(popoverElement)
         .addTo(map);
-    });
+    };
+
+    map.on('click', `${activeParameter}-circles`, openPopup);
+    map.on('click', `${activeParameter}-squares`, openPopup);
   }, [isAllLocations, selectedLocations]);
 
   useEffect(() => {
-    if (country && map.getLayer(`${activeParameter}-layer`)) {
-      map.setFilter(`${activeParameter}-outline`, ['==', 'country', country]);
-      map.setFilter(`${activeParameter}-layer`, ['==', 'country', country]);
-
-      return () => {
-        map.setFilter(`${activeParameter}-outline`, null);
-        map.setFilter(`${activeParameter}-layer`, null);
-      };
+    if (country && map.getLayer(`${activeParameter}-circles`)) {
+      map.setFilter(`${activeParameter}-square-outline`, squaresCountryFilter);
```

Comment:
![image](https://user-images.githubusercontent.com/19439427/104229412-421cd200-541a-11eb-8a66-41893a2e084b.png)
I think this is not working (at least on location dashboard) because country has not been passed to `MeasurementsLayer`.  We should pass country [here](https://github.com/openaq/openaq.org/blob/feature/472-map-symbols/app/assets/scripts/views/location/nearby-locations.js#L38)

Reply:
I intentionally did not filter by country on the location dashboard, I thought we'd want to see all surrounding stations even when across the border. Usually the map would be zoomed in to the selected location and highlight it. Should we limit the zoom range on the location page to not be able to get to the view you shared above? Its a question for @rwegener2 and @tania-pires I'd say